homeassistant/components/sense/binary_sensor.py

```diff
@@ -96,20 +63,27 @@ def sense_to_mdi(sense_icon):
 class SenseDevice(BinarySensorDevice):
```

Comment:
We should remove the watts from the binary sensor attributes.

Reply:
This was done